website/docs/r/windows_virtual_machine.html.markdown

```diff
@@ -143,8 +147,14 @@ The following arguments are supported:
 
 -> **NOTE:** This can only be configured when `priority` is set to `Spot`.
 
+* `patch_assessment_mode` -  (Optional) Specifies the mode of VM Guest patch assessment for the virtual machine. Possible values are `ImageDefault` or `AutomaticByPlatform`.
+
+-> **NOTE:** If `patch_assessment_mode` is set then the `provision_vm_agent` must also be set to `true`.
+
 * `patch_mode` - (Optional) Specifies the mode of in-guest patching to this Windows Virtual Machine. Possible values are `Manual`, `AutomaticByOS` and `AutomaticByPlatform`. Defaults to `AutomaticByOS`.
 
+-> **NOTE:** If `patch_mode` is set to `AutomaticByPlatform` then `provision_vm_agent` must also be set to `true`.
+
```

Comment:
are these both preview-only features?

Reply:
Per the service team only patch assessment is in preview, the other two have GA'ed... 🤷‍♂️